modules/custom/az_paragraphs/src/Plugin/paragraphs/Behavior/AZDefaultParagraphBehavior.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+namespace Drupal\az_paragraphs\Plugin\paragraphs\Behavior;
+
+use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\paragraphs\Entity\Paragraph;
+use Drupal\paragraphs\ParagraphInterface;
+use Drupal\paragraphs\ParagraphsBehaviorBase;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Provides a base behavior for Quickstart Paragraph Defaults.
+ *
+ * @ParagraphsBehavior(
+ *   id = "az_default_paragraph_behavior",
+ *   label = @Translation("Default Quickstart Paragraph Behavior"),
+ *   description = @Translation("Provides default Quickstart paragraph settings."),
+ *   weight = 0
+ * )
+ */
+class AZDefaultParagraphBehavior extends ParagraphsBehaviorBase {
```

Comment:
Nitpick:  Would it be better to name our behavior `AZDefaultParagraphsBehavior` (with an `s`).  It seems that within the Paragraphs module, behaviors are always referred to as Paragraphs Behaviors.

Reply:
I agree, that seems better. Updated to change the name to `AZDefaultParagraphsBehavior`.